image.yaml

```diff
@@ -0,0 +1,31 @@
+- name: operator-builder
+  version: 1.0.0
+  from: golang:1.19
+  description: Builder Image for the Operator
+
+  modules:
+    repositories:
+      - path: modules
+    install:
+      - name: org.kie.kogito.goModDownload
+      - name: org.kie.kogito.app.builder
+- name: kogito-serverless-operator
```

Comment:
```suggestion


- name: kogito-serverless-operator
```

Reply:
not mandatory, just to make easier to find where one image definition ends and the other starts.